lib/perl/Genome/Qc/Command/BuildMetrics.pm

```diff
@@ -0,0 +1,70 @@
+package Genome::Qc::Command::BuildMetrics;
+
+use strict;
+use warnings;
+
+use Genome;
+use Data::Dumper;
+
+class Genome::Qc::Command::BuildMetrics {
+    is => 'Command::V2',
+    has => [
+        builds => {
+            is => 'Genome::Model::Build',
+            is_many => 1,
+            shell_args_position => 1,
+            doc => 'The builds to report QC metrics for.',
+        },
+    ]
+};
+sub help_brief {
+    'A command to print the QC result metrics for input builds.'
+}
+
+sub help_synopsis {
+    'Dump QC result metrics from the database to tab-delimited output.'
+}
+
+sub help_detail{
+    return <<"EOS"
+The QC framework stores result metrics in the database for each QC result.  This tool will dump the instrument data QC result metrics for all input builds.  A tab-delimited output of all QC metrics along with build id and instrument data id are output to the terminal.
+EOS
+}
+
+sub execute {
+    my $self = shift;
+
+    my @metrics;
+    for my $build ($self->builds) {
+        my @qc_results = grep {$_->isa('Genome::Qc::Result')} $build->results;
+        for my $qc_result (@qc_results) {
+            my $as = $qc_result->alignment_result;
+            my @instrument_data = $as->instrument_data;
+            if (@instrument_data > 1) {
+                $self->error_message('Please add support for merged alignment results and multiple instrument data QC results!');
```

Comment:
You could easily support this by putting the instrument data ids in a comma-separated list (for example) - I think everything else is generic enough that you wouldn't need to change anything.